hota/Mods/HotA banks/Content/config/banks/banks/pirateCavern.json

```diff
@@ -20,19 +20,18 @@
 					"ambient": ["banks/LOOPPCAV.wav"]
 				},
 				"rmg" : {
-					"value"		: 2500,
+					"value"		: 3500,
 					"rarity"	: 100
 				},
 				"levels":
 				[
 					{
 						"chance": 30,
 						"guards": [
-							{ "amount": 10, "type" : "pirate" },
-							{ "amount": 10, "type" : "pirate" },
-							{ "amount": 10, "type" : "corsair", "upgradeChance": 50 },
-							{ "amount": 10, "type" : "pirate" },
-							{ "amount": 10, "type" : "corsair" }
+							{ "amount": 20, "type" : "pirate", "upgradeChance": 50 },
```

Comment:
According to: https://heroes.thelazy.net//index.php/Pirate_Cavern old guards are correct. In this variant there are 40 guards dividen into 4 stacks.

Reply:
Fixed. Now bank has 4 stacks with 10/20/30/40 guards, which matches HotA